app/src/main/java/com/infomaniak/mail/ui/newMessage/RecipientFieldView.kt

```diff
@@ -214,7 +214,7 @@ class RecipientFieldView @JvmOverloads constructor(
     private fun setTextInputListeners() = with(binding.textInput) {
 
         fun performContactSearch(text: CharSequence) {
-            if ((text.trim().count()) > 0) {
+            if (text.isNotEmpty()) {
```

Comment:
Why remove the .trim()?

Reply:
It was generating a warning and both are the same.